projects/gen-ai.md

```diff
@@ -0,0 +1,143 @@
+- **Cost Tracking**: Monitoring operational costs like computation time and API calls for resource optimization.
+- **AI Safety**: Monitoring for ethical LLM use, including privacy compliance, bias detection, and secure data handling.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
+For Metrics: Tracking model performance by collecting and correlating data like model names, versions, parameters, latency, and evaluation scores.
+
+### Current challenges
+
+Existing GenAI observability projects are driven independently by individual vendors which don't have a common space to collaborate on common instrumentation libraries.
+
+[OpenLLMetry](https://github.com/traceloop/openllmetry) project was created with the intention [to create space under CNCF](https://github.com/cncf/sandbox/issues/67) and gave a start to OTel [LLM Semantic Convention](https://github.com/open-telemetry/community/blob/main/projects/llm-semconv.md) project.
+
+OpenLLMetry provides general-purpose OTel instrumentations for GenAI clients, frameworks and related components; they are governed by the [Traceloop](https://www.traceloop.com/docs/introduction).
+
+By providing GenAI community with the space under OpenTelemetry we will be able to start hosting vendor-agnostic instrumentation libraries (such as WIP one [OpenAI Python](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2759)) and eventually we will be able to move popular OpenLLMetry-owned `opentelemetry-instrumentation-*` libraries to OTel (or to upstream libraries).
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
```

Comment:
Might be worth calling out what can be delivered in short/mid term to get it to a certain milestone, and then what can be considered later if this SIG is to run for an indefinite amount of time (e.g. other languages apart from Python and Node)

Reply:
good point, updated